app/models/billing_expense.rb

```diff
@@ -11,7 +11,13 @@ class BillingExpense < ApplicationRecord
 
   validates :amount, inclusion: { in: AMOUNT }
 
-  def self.amount_collection
-    AMOUNT.map(&:to_s)
+  def billing_hours
+    hours = Hour.where(billing_expense: id).sum(&:hours)
```

Comment:
```ruby
Hour.where(billing_expense: id).sum(&:hours)
Hour.where(billing_expense: id).sum(:hours)
```
whats the `&` for?

Reply:
https://github.com/panter/aoz-003/pull/211#discussion_r137497771